From: Ana Guerrero Lopez <aguerrero@suse.com>
Date: 2018-01-21 16:00:00 +0200
Subject: do not install examples scripts under /usr/bin
Upstream: tbd

Example scripts are installed under /usr/bin and it's prefered to keep
them in an examples directory.

--- a/setup.py
+++ b/setup.py
@@ -103,7 +103,7 @@
     'classifiers': [_f for _f in classifiers.split("\n") if _f],
     'url': home_page,
     'download_url': dwnl_url,
-    'scripts': scripts,
+#    'scripts': scripts,
     'data_files': data_files,
     'install_requires': ['SQLAlchemy', 'lxml'],
     'extras_require': {
